lib/js/components/Badges/Badge/Badge.stories.ts

```diff
@@ -0,0 +1,39 @@
+// Badge.stories.ts
+
```

Comment:
```suggestion
```

Reply:
It was suggestion in SB tutorials to add it, but it do nothing. So we can remove it.

Reply:
it's there only to tell you in which file you're, because in web code snippets you usually don't see filename